pkg/gui/files_panel.go

```diff
@@ -80,28 +74,28 @@ func (gui *Gui) handleFilePress(g *gocui.Gui, v *gocui.View) error {
 func (gui *Gui) handleAddPatch(g *gocui.Gui, v *gocui.View) error {
 	file, err := gui.getSelectedFile(g)
 	if err != nil {
-		if err == errNoFiles {
+		if err == gui.Errors.ErrNoFiles {
 			return nil
 		}
 		return err
 	}
 	if !file.HasUnstagedChanges {
-		return gui.createErrorPanel(g, "File has no unstaged changes to add")
+		return gui.createErrorPanel(g, gui.Tr.SLocalize("FileHasNoUnstagedChanges"))
 	}
 	if !file.Tracked {
-		return gui.createErrorPanel(g, "Cannot git add --patch untracked files")
+		return gui.createErrorPanel(g, gui.Tr.SLocalize("CannotGitAdd"))
 	}
 	sub, err := gui.GitCommand.AddPatch(file.Name)
 	if err != nil {
 		return err
 	}
 	gui.SubProcess = sub
-	return ErrSubProcess
+	return nil
```

Comment:
can we make this `return gui.Errors.ErrSubProcess`

Reply:
I'll test that out and see if the go checker doesn't spam me with errors